webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardReverse.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <v-card class="peopleCardItem d-flex flex-column" flat>
+  <v-card 
+    class="peopleCardItem d-flex flex-column" 
+    flat
+    :outlined="!isMobile">
```

Comment:
Variablilized properties first (best practices)

```suggestion
    :outlined="!isMobile"
    class="peopleCardItem d-flex flex-column" 
    flat>
```

Reply:
done!